WeakAuras/WeakAuras.lua

```diff
@@ -944,7 +944,7 @@ do -- Archive stuff
     return Archivist
   end
 
-  function WeakAuras.LoadFromArchive(storeType, storeID)
+  function Private.LoadFromArchive(storeType, storeID)
```

Comment:
This breaks a couple auras i've seen made in the discord. But I intend to obsolete the use cases which needed that anyways.

Reply:
Well, I'll drop the patch then.